From 46df3b2a74dcdea4c91e5d0b4971fa08781721d0 Mon Sep 17 00:00:00 2001
From: Kevin Chen <kevin_chen@aspeedtech.com>
Date: Fri, 12 Sep 2025 17:12:27 +0800
Subject: [PATCH 28/32] dts: ast2700-ibex: Rename eDAF DDR mode node for
 consistency

Rename the eDAF DDR mode node from 'edaf0' to 'edaf1' in the reserved
memory description for improved naming consistency and clarity.

No functional changes introduced.

Change-Id: Ie9cf340489ec525aafbd15a505457197c3fd3b30
Signed-off-by: Kevin Chen <kevin_chen@aspeedtech.com>
(cherry picked from commit 22c9bb405867e93860b4f3cdd1abc94f9cecf4a5)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 arch/riscv/dts/ast2700-ibex-reserved-mem.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/dts/ast2700-ibex-reserved-mem.dtsi b/arch/riscv/dts/ast2700-ibex-reserved-mem.dtsi
index 0b0001f71a..cd619881dc 100644
--- a/arch/riscv/dts/ast2700-ibex-reserved-mem.dtsi
+++ b/arch/riscv/dts/ast2700-ibex-reserved-mem.dtsi
@@ -69,7 +69,7 @@ espi1_mmbi_memory: espi1-mmbi-memory@a4000000 {
 };
 
 // This area used for eDAF DDR mode even in MIX mode or HW mode(enabled in SPL)
-edaf0: safs-ddr-mode@a8000000 {
+edaf1: edaf1-memory@a8000000 {
 	reg = <0xa8000000 0x4000000>;
 	no-map;
 	bootph-all;
-- 
2.34.1

